panic: runtime error: invalid memory address or nil pointer dereferernce

This was on HEAD, I think july 5th, but before the last 
458d7edd73046ab806a5d1aacb26989dfa2ce03c "Run go fix across project" commit. 

I just pulled and re installed, so hopefully it's fixed, but wanted to let you know!


```
2018-07-09 08:07:09.684 [INF] WLLT: Inserting unconfirmed transaction 69d66799419b775aa5b9671292709bf79e312f0b39af692fb7e526e2de74c0af

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x80a87d]

goroutine 50 [running]:
github.com/decred/dcrwallet/wallet.makeTicket(0x1135b80, 0x0, 0xc42027f730, 0xd7c020, 0xc4201fc260, 0xd7c020, 0xc4201ea540, 0x24675d341, 0x0, 0x0, ...)
        /home/gabe/src/github.com/decred/dcrwallet/wallet/createtx.go:750 +0x1bd
github.com/decred/dcrwallet/wallet.(*Wallet).purchaseTickets(0xc4201d8000, 0xba82b8, 0x16, 0x0, 0x2a2d6f5da, 0x0, 0xd7c020, 0xc4201fc260, 0x0, 0x1, ...)
                /home/gabe/src/github.com/decred/dcrwallet/wallet/createtx.go:1115 +0x1122
github.com/decred/dcrwallet/wallet.(*Wallet).txCreator(0xc4201d8000)
                /home/gabe/src/github.com/decred/dcrwallet/wallet/wallet.go:1016 +0x38e
created by github.com/decred/dcrwallet/wallet.(*Wallet).Start
                /home/gabe/src/github.com/decred/dcrwallet/wallet/wallet.go:451 +0xd8
```

--- a/wallet/createtx.go
+++ b/wallet/createtx.go
@@ -747,7 +747,7 @@ func makeTicket(params *chaincfg.Params, inputPool *extendedOutPoint, input *ext
 		mtx.AddTxIn(txIn)
 	}
 
-	txIn := wire.NewTxIn(input.op, inputPool.amt, []byte{})
+	txIn := wire.NewTxIn(input.op, input.amt, []byte{})
 	mtx.AddTxIn(txIn)
 
 	// Create a new script which pays to the provided address with an

